indigo_api/models/works.py

```diff
@@ -170,6 +170,19 @@ def repeal(self):
     def place(self):
         return self.locality or self.country
 
+    @property
+    def amended(self):
+        if self.amendments.all():
+            return True
+
+    @property
+    def most_recent_amendment_year(self):
```

Comment:
It's far cheaper to do the sorting in the db: `self.amendments.order_by('-date').first()` or something similar.

Reply:
Ditto property comment from above.